wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/ConversationSystemMessageCellDescription.swift

```diff
@@ -210,6 +210,14 @@ final class ConversationSystemMessageCellDescription {
         case .invalid:
             let unknownMessage = UnknownMessageCellDescription()
             return [AnyConversationMessageCellDescription(unknownMessage)]
+
+        case .mlsMigrationMLSNotSupportedSelfUser, .mlsMigrationMLSNotSupportedOtherUser:
```

Comment:
question: Maybe `case .invalid` should stay the last case? Not really important for me

Reply:
I will move it at the end.